### DISCUSSION TITLE
Class suggestions for liquid snippets

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#11209] [Help] [answered]
Hi guys,

Anyone knows if it's possible to make Tailwind CSS IntelliSense work with liquid snippets? Otherwise I would like to see this feature.

For example:

```liquid
{%- render 'snippet', class: 'suggestions-here' -%}
```

I tried to play with `tailwindCSS.experimental.classRegex`, but without success. I think because this is only working on html tags or some kind of component declaration in e.g. React.

Thanks!

**Comment by wongjn** [comment, marked as answer]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659), something like:

```json
"tailwindCSS.experimental.classRegex": [
  "{%- render 'snippet', class: '([^']*)"
]
```

You may also need to configure [`tailwindCSS.includeLanguages`](https://github.com/tailwindlabs/tailwindcss-intellisense#tailwindcssincludelanguages).

**Comment by ghost** [comment]
Hi @wongjn,

Well... That was easy 😂. I also don't have to configure the language. I already did this through `emmit.includeLanguages`
, which is probably enough.

Thanks for the quick response!
